libknet/bindings/rust/build.rs

```diff
@@ -0,0 +1,30 @@
+extern crate pkg_config;
+
+fn main() {
+    if let Err(e) = pkg_config::probe_library("libknet") {
+	match e {
+	    pkg_config::Error::Failure { .. } => panic! (
```

Comment:
I would assume we will need to port this one to the build system...

Reply:
not sure what you mean by that. build.rs is the build "script" that cargo uses (mostly encountered for stuff like crates needing to generate bindings, or emit special linker/compiler flags). in this case, the `probe_library` call will (if successful) emit the linking flags on STDOUT, which cargo will pick up and pass along to rustc.

Reply:
I mean that pkg-config is not really usable till the library is installed on the target system. For internal building we might need something else.